src/InsetOrderOptimizer.cpp

```diff
@@ -216,6 +218,25 @@ BinJunctions InsetOrderOptimizer::variableWidthPathToBinJunctions(const Variable
             }
         }
     }
+
+    if (alternate_inset_direction) {
+        bool alternate_inset_print_direction = reverse_initial_inset;
+        for (PathJunctions &paths: insets) {
+            for (LineJunctions &line: paths) {
+                if (alternate_inset_print_direction) {
+                    int left = 0;
+                    int right = line.size() - 1;
+                    while (left < right) {
+                        swap(line[left], line[right]);
+                        left++;
+                        right--;
```

Comment:
can be shortened to:
```suggestion
                        swap(line[left++], line[right--]);
```
Since the `++` and `--` values are in-/decremented after they are used. If you use the `++left` they are incremented before the value is used.
But I'm okay if you leave it like this.

Reply:
I shortened it even further by using `std::reverse` instead of this custom implementation.